map/map_loader/src/lanelet2_map_loader/lanelet2_map_loader_node.cpp

```diff
@@ -93,6 +100,21 @@ lanelet::LaneletMapPtr Lanelet2MapLoaderNode::load_map(
     if (errors.empty()) {
       return map;
     }
+  } else if (lanelet2_map_projector_type == "local") {
+    // Use MGRSProjector as parser
+    lanelet::projection::MGRSProjector projector{};
+    const lanelet::LaneletMapPtr map = lanelet::load(lanelet2_filename, projector, &errors);
+
+    // overwrite local_x, local_y
+    for (lanelet::Point3d point : map->pointLayer) {
+      if (point.hasAttribute("local_x")) {
+        point.x() = point.attribute("local_x").asDouble().value();
+      }
+      if (point.hasAttribute("local_y")) {
+        point.y() = point.attribute("local_y").asDouble().value();
+      }
+    }
+    return map;
```

Comment:
Here we only detect local attributes. If it is detected, override lanelet2_map_projector_type to local.

Reply:
But I think most of the map will have local attributes if is generated by VMB.
If we only detect local attributes, most of the map will be local

Reply:
If the map has a local attribute, it cannot be MGRS or UTM. Because there is no guarantee that the local coordinates will match the latitude and longitude. Therefore maps containing local attributes cannot support global pose. I think the VMB should not output redundant local attributes.

Reply:
We decided to use local type in https://github.com/autowarefoundation/autoware/issues/3427.